.coveragerc

```diff
@@ -1160,6 +1160,8 @@ omit =
     homeassistant/components/volumio/browse_media.py
     homeassistant/components/volumio/media_player.py
     homeassistant/components/volvooncall/*
+    homeassistant/components/vorwerk/__init__.py
+    homeassistant/components/vorwerk/vacuum.py
```

Comment:
It is not compulsory, but since new integrations can take a while to get merged, I suggest that you take the opportunity to move to 100% coverage whilst the reviews are in progress.
```suggestion
```

Reply:
I will improve that over Christmas.

Reply:
What is there to do? Just removing these lines? I want to help.